mailpoet/lib/Cron/Workers/Scheduler.php

```diff
@@ -187,21 +187,31 @@ public function processPostNotificationNewsletter($newsletter, SendingTask $queu
     if (empty($subscribersCount)) {
       $this->loggerFactory->getLogger(LoggerFactory::TOPIC_POST_NOTIFICATIONS)->info(
         'post notification no subscribers',
-        ['newsletter_id' => $newsletter->id, 'task_id' => $queue->taskId]
+        ['newsletter_id' => $newsletter->id, 'task_id' => $queue->taskId, 'segment_ids' => $segments]
       );
       return $this->deleteQueueOrUpdateNextRunDate($queue, $newsletter);
     }
 
     // create a duplicate newsletter that acts as a history record
-    $notificationHistory = $this->createPostNotificationHistory($newsletterEntity);
+    try {
+      $notificationHistory = $this->createPostNotificationHistory($newsletterEntity);
+    } catch (\Exception $exception) {
+      $this->loggerFactory->getLogger(LoggerFactory::TOPIC_POST_NOTIFICATIONS)->error(
+        'creating post notification failed',
```

Comment:
I think 'creating post notification failed' should be 'creating post notification history failed'.